ingestion/functions/README.md

```diff
@@ -248,4 +248,5 @@ Here is an overview of parsers written so far and some details about the data th
 | Estonia                     | [code](https://github.com/globaldothealth/list/tree/main/ingestion/functions/parsing/estonia)        |                                                                                                                                                                                                                                                                                                     | #502 |
 | Amapa, Brazil               | [code](https://github.com/globaldothealth/list/tree/main/ingestion/functions/parsing/brazil_amapa)   | There are no patient ID/case ID in the raw API so we aren't able to dedupe. There are two files in the source for Amapa, one for confirmed cases and one for confirmed deaths; some of these cases may also be deaths but without patient IDs we are unable to confirm.                             | #495 |
 | Paraiba, Brazil             | [code](https://github.com/globaldothealth/list/tree/main/ingestion/functions/parsing/brazil_paraiba) | There are no patient ID/case ID in the raw API so we aren't able to dedupe. This data only includes deaths from Covid-19 so the outcome of all cases will be death. There is no date of confirmation.                                                                                               | #499 |
-| Peru                        | [code](https://github.com/globaldothealth/list/tree/main/ingestion/functions/parsing/peru)           | Assuming PR = prueba rapida (rapid serological test) and PCR = PCR test                                                                                                                                                                                                                             | #484 |
\ No newline at end of file
+| Peru                        | [code](https://github.com/globaldothealth/list/tree/main/ingestion/functions/parsing/peru)           | Assuming PR = prueba rapida (rapid serological test) and PCR = PCR test                                                                                                                                                                                                                             | #484 |
+| Taiwan                      | [code](https://github.com/globaldothealth/list/tree/main/ingestion/functions/parsing/taiwan)         | No per-case ID so unable to dedupe.                                                                                                                                                                                                                                                                 | #517 |
```

Comment:
How does it work if we're unable to dedup? Only runs once per day? Could you explain it here

Reply:
it's explained a bit up:
https://github.com/globaldothealth/list/blob/main/ingestion/functions/README.md#writing-a-parser-where-deduplication-of-patients-cannot-be-done